common/web/gesture-recognizer/src/test/auto/headless/gesturePath.js

```diff
@@ -92,8 +94,7 @@ describe("GesturePath", function() {
   "wasCancelled": true
 }
       `.trim();
-
-      assert.equal(JSON.stringify(serializationObj, null, 2), SERIALIZATION_TO_MATCH);
+      assert.equal(JSON.stringify(serializationObj, (key, value) => key == 'stats' ? undefined : value, 2), SERIALIZATION_TO_MATCH);
```

Comment:
The object returned by `.toJSON` can be further processed; here, we filter out the `stats` field via JSON.stringify replacer.  (It wasn't there when the original test was written, so we get rid of it to match the original expectation.)